src/cim/messages.test.ts

```diff
@@ -7,8 +7,12 @@ import { WSManErrors } from '../WSMan'
 import { Classes, Methods, Messages } from '.'
 
 describe('CIM Tests', () => {
+  beforeEach(() => {
+    messageId = 0
+    cimClass = new Messages()
+  })
   let messageId = 0
-  const cimClass = new Messages()
+  let cimClass = new Messages()
```

Comment:
seems wierd to define these even though they are in the beforeEach, i would move them up a level and _not_ define them and let the beforeEach do that.